templates/BoaJavaHadoop.stg

```diff
@@ -80,7 +82,10 @@ public class <name> extends boa.runtime.BoaRunner {
 
 		if (id > 0)
 			configuration.setInt("boa.hadoop.jobid", id);
-		jb.submit();
```

Comment:
This change is completely unnecessary.  Notice right below the option for 'block'.  That is essentially what you are doing here.